azure/database/postgres/main.tf

```diff
@@ -0,0 +1,46 @@
+resource "azurerm_postgresql_flexible_server" "main" {
+  name                   = local.name
+  resource_group_name    = var.resource_group_name
+  location               = var.region
+  version                = var.postgres_version
+  delegated_subnet_id    = azurerm_subnet.main.id
+  private_dns_zone_id    = azurerm_private_dns_zone.main.id
+  administrator_login    = var.administrator_login
+  administrator_password = var.administrator_password
+  create_mode            = var.create_mode
+  storage_mb             = var.storage_mb
+  storage_tier           = var.storage_tier
+  sku_name               = var.sku_name
+
```

Comment:
should we explicitly set [backup_retention_days](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/postgresql_flexible_server#backup_retention_days) to `7` to ensure we do have this configured? (future versions of the TF module might change this)